Tasks/Common/webdeployment-common-v2/fileTransformationsUtility.ts

```diff
@@ -17,12 +17,8 @@ export function fileTransformations(isFolderBasedDeployment: boolean, JSONFiles:
                 console.log(tl.loc('AutoParameterizationMessage'));
             }
         }
-        var environmentName = tl.getVariable('Release.EnvironmentName');
```

Comment:
do we use this function in FileTransform task? If not we might not want to remove this.